src/methods/modular_programs/easy/q6/Century.java

```diff
@@ -0,0 +1,40 @@
+package methods.modular_programs.easy.q6;
+
+/*
+Write a Java Program that asks users for the year they were born, then returns what century that year is in. The program
```

Comment:
Change the question text to:
`Write a Java Program that asks users for the year they were born, then prints the century the individual was born. The program should have a method called centuryCalc, which determines the century and works for any century from the 1500s to the 2000's. If the input is in any other century, return "Unknown Century".
`

Reply:
We should also maybe add a note that the 1500s is the 16th century and so on